import_export/resources.py

```diff
@@ -438,18 +438,18 @@ def validate_instance(self, instance, import_validation_errors=None, validate_un
         if errors:
             raise ValidationError(errors)
 
-    def save_instance(self, instance, using_transactions=True, dry_run=False):
+    def save_instance(self, instance, using_transactions=True, dry_run=False, is_create=False):
```

Comment:
I'd say we need to have `is_create` as a mandatory argument, don't use `False` as default. Like the previous code was explicitly checking the `pk`, we should get a explicit value for `is_create`. What do you think?

Reply:
Agreed - have made this change.